DeviceAgent.xcodeproj/project.pbxproj

```diff
@@ -3737,7 +3737,8 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Distribution: Karl Krukow (FYD86LA7RE)";
```

Comment:
Not sure this should be here. Generic iPhone developer seems nicer. It depends on how this is managed, but fastlane actually can cover all of this.

Reply:
Also, who is Karl Krukow?

Reply:
Karl is the original author of Calabash iOS and Android.

He founded a company in Denmark called Lesspainful which was bought by Xamarin. Xamarin was bought by Microsoft.

We continue to use his credentials for signing.

We cannot use a generic developer ID.

We don't want to use fastlane.